scipy/stats/meson.build

```diff
@@ -120,7 +120,8 @@ _stats_gen_pyx = custom_target('_stats_gen_pyx',
     'nbinom_ufunc.pyx',  # 5 (S_B)
     'templated_pyufunc.pxd',  # 6 (S_B)
     'unuran_wrapper.pyx',  # 7 (used in stats/_unuran)
-    'ncf_ufunc.pyx'  # 8 (S_B)
+    'ncf_ufunc.pyx',  # 8 (S_B)
+    'ncx2_ufunc.pyx'  # 9 (S_B)
```

Comment:
I was going to suggest it last time, but maybe going forward we should include `,` at the end of this line in anticipation of additions? +0.5, so if you're <=-0, we can leave it alone.

Reply:
I love trailing commas -- I was just following the style already present in the file.  I will make the switch